Readme.md

```diff
@@ -2,10 +2,14 @@ A tiny utility to post current directory contents to http://plnkr.co/edit
 
 ## Anonymous Usage
 
+0. Add `bin` project folder to your path
```

Comment:
Install globally works OK on windows. But unfortunately, if I don't want to install plunk globally (for better or worse), I have no other option than to use PATH (see more in the other comment).